tests/bsim/bluetooth/mesh/tests_scripts/suspend/suspend_disable_resume.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+# Copyright 2023 Nordic Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
+source $(dirname "${BASH_SOURCE[0]}")/../../_mesh_test.sh
+
+# Test that periodic publication is stopped when suspending Mesh and disabling
+# Bluetooth, and that it is started again when Bluetooth is re-enabled and Mesh is resumed.
+# The test will fail if no publication is received, or if a publication is received when Mesh
+# is disabled.
+#
+# Test procedure:
```

Comment:
This test procedure might be missing what is checked during and after suspending and resuming mesh. Same for the other test script

Reply:
Added some more info.